fmt/format.h

```diff
@@ -1978,11 +1987,11 @@ class PrintfFormatter : private FormatterBase {
  */
 template <typename Impl, typename Char>
 class BasicArgFormatter : public internal::ArgFormatterBase<Impl, Char> {
- private:
+private:
```

Comment:
Please preserve the original formatting that follows [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html) here and in other places.